plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/azure/AzureBlobFileSystemExchangeStorage.java

```diff
@@ -296,11 +300,14 @@ public AzureExchangeStorageReader(
                 BlobServiceAsyncClient blobServiceAsyncClient,
                 List<ExchangeSourceFile> sourceFiles,
                 int blockSize,
-                int maxPageStorageSize)
+                int maxPageStorageSize,
```

Comment:
move `metricsBuilder` below `sourceFiles` ? I prefer to keep objects together and primitive values separately and do not mix one with each other

Reply:
I do not think it is generally beneficial - I would rather try to group logically. But I do not care in this case so moved.